src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -144,3 +144,11 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
 register_cli_argument('appservice web source-control', 'branch', help='the branch name of the repository')
 register_cli_argument('appservice web source-control', 'repository_type', help='repository type', default='git', **enum_choice_list(['git', 'mercurial']))
 register_cli_argument('appservice web source-control', 'git_token', help='git access token required for auto sync')
+
+register_cli_argument('functionapp', 'name', configured_default='web',
+                      arg_type=name_arg_type, completer=get_resource_name_completion_list('Microsoft.Web/sites'), id_part='name',
+                      help="name of the web. You can configure the default using 'az configure --defaults web=<name>'")
```

Comment:
Let us remove the `You can configure the default...`, and `configured_default` since functionapp is to be made significantly different from the regular webapp.